### DISCUSSION TITLE
Support the vue?

### DISCUSSION THREAD

**Discussion by white-007** [DouyinFE/semi-design#162] [Ideas]
### Which Component Need Enhancement 期望新增功能的组件
- Vuejs

### What does the proposed API look like 期望支持的API
Vuejs

### Feature Description 功能描述
Vuejs

### Additional information 补充说明
Vuejs

**Comment by hmwl** [comment]
目前只支持react吗？希望支持vue

**Comment by xxxxue** [comment]
希望支持Solid.js

**Comment by shijiatongxue** [comment]
Vue has a relatively low priority internally. If necessary, we may rely on external students to provide it. We provide support and guidance. Thanks!

Vue 在我们内部优先级比较低，后续如果有必要，有可能靠外部同学来提供，我们提供支持和指导。
